.changeset/swift-rabbits-appear.md

```diff
@@ -0,0 +1,7 @@
+---
+"@latticexyz/explorer": patch
```

Comment:
this should probably be `minor`

Reply:
Updated. Does it require version bump? Based on my understanding no because none of the existing packages are affected

Reply:
we currently have all MUD packages tied together in their versioning/release cycle, so in the context of MUD it's a new (and big!) feature, so I think tagging it as a bump to v2.2 makes the most sense?

Reply:
Agreed on v2.2!